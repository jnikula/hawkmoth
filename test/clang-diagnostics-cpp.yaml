domain: cpp
directive-arguments:
  - clang-diagnostics.c
directive-options:
  clang:
    - -xc++
