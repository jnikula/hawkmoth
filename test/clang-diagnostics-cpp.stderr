ERROR: clang-diagnostics.c:4: ISO C++ forbids forward references to 'enum' types
